test/e2e/basic_publish.test.ts

```diff
@@ -47,6 +46,5 @@ describe("publish a message", () => {
     await eventually(async () => {
       expect((await rabbit.getQueueInfo(stream)).messages).eql(100)
     }, 10000)
-    await connection.close()
```

Comment:
Why do you remove this code?

Reply:
the
```
afterEach(() => connection.close())
```

closes the connection 2 times, giving an error now that the close on the rabbit level is done. do you think we have to handle that differently?

Reply:
as above